index.bs

```diff
@@ -188,11 +189,13 @@ Navigation response {#navigation-response}
 
 At [=process a navigate response=], after step 7 call [$update the Client Hints set$] with the [=relevant settings object=] and |response| as inputs.
 
-Worker initialization {#worker-init}
+Service Worker initialization {#service-worker-init}
 -----------
 At <a href="https://html.spec.whatwg.org/multipage/workers.html#set-up-a-worker-environment-settings-object">set up a worker environment settings object</a>,
 after step 6, add the following step:
-1. Set |settingsObject|'s [=environment settings object/client hints set=] to be a [=set/clone=] of <var ignore>outside settings</var>' [=environment settings object/client hints set=].
+<ol>
+ <li>If worker global scope is a {{ServiceWorkerGlobalScope}} object, then set |settingsObject|'s [=environment settings object/client hints set=] to be a [=set/clone=] of <var ignore>outside settings</var>' [=environment settings object/client hints set=].
```

Comment:
```suggestion
 <li>If <var ignore>worker global scope</var> implements {{ServiceWorkerGlobalScope}}, then set |settingsObject|'s [=environment settings object/client hints set=] to be a [=set/clone=] of <var ignore>outside settings</var>' [=environment settings object/client hints set=].
```

Reply:
Also, not new in this PR, but where does `settingsObject` come from? It seems to be undefined.

Reply:
Should have been |setting object|, it’s defined in https://html.spec.whatwg.org/multipage/workers.html#set-up-a-worker-environment-settings-object